client/extensions/woocommerce/app/reviews/review-card.js

```diff
@@ -29,43 +33,141 @@ class ReviewCard extends Component {
+			<Button
+				borderless
+				className="reviews__action-collapse"
+				onClick={ isExpanded ? this.toggleExpanded : noop }
+			>
+				<Gridicon icon="chevron-down" />
+			</Button>
+		);
+	}
+
+	renderGravatar() {
 		const { review } = this.props;
-		const classes = classNames( 'reviews__card', {
-			'is-approved': review.approved,
-			'is-unapproved': ! review.approved,
-		} );
 		const author = {
 			avatar_URL: review.avatar_urls[ 48 ],
 			display_name: review.name,
 		};
+		return <Gravatar user={ author } />;
+	}
 
+	renderProductImage() {
+		const { review } = this.props;
 		const productImageClasses = classNames( 'reviews__product', { 'is-placeholder': ! review.product.image } );
+		return (
+			<div className={ productImageClasses }>
+				{ review.product.image && ( <img src={ review.product.image } /> ) }
+			</div>
+		);
+	}
 
+	renderActionsBar() {
+		const { review, currentStatus } = this.props;
 		return (
-			<Card className={ classes }>
-				<div className={ classNames( 'reviews__header', 'is-preview' ) } >
+			<div className={ classNames( 'reviews__header' ) } >
+				<ReviewActionsBar review={ review } currentStatus={ currentStatus } />
+				{ this.renderToggle() }
+			</div>
+		);
+	}
+
+	renderPreview() {
+		const { review } = this.props;
+		return (
+			<div
+				className={ classNames( 'reviews__header', 'is-preview' ) }
+				onClick={ this.toggleExpanded }
+			>
+				<div className="reviews__author-gravatar">
+					{ this.renderGravatar() }
+				</div>
+				<div className="reviews__info">
+					<div className="reviews__author-name">
+						{ review.name }
+						{ review.verified && <Gridicon icon="checkmark-circle" size={ 18 } /> }
+					</div>
+					<div className="reviews__date">{ humanDate( review.date_created_gmt + 'Z' ) }</div>
+				</div>
+				<AutoDirection>
+					<div className="reviews__content">
+						{ decodeEntities( stripHTML( review.review ) ) }
+					</div>
+				</AutoDirection>
+				<div className="reviews__rating">
+					<Rating rating={ ( review.rating / 5 ) * 100 } />
+				</div>
+				{ this.renderProductImage() }
+				{ this.renderToggle() }
+			</div>
+		);
+	}
+
+	renderExpandedCard() {
+		const { review, translate } = this.props;
+		return (
+			<div className="reviews__expanded-card">
+				<div className="reviews__expanded-card-details">
 					<div className="reviews__author-gravatar">
-						<Gravatar user={ author } />
+						{ this.renderGravatar() }
 					</div>
+
 					<div className="reviews__info">
 						<div className="reviews__author-name">{ review.name }</div>
 						<div className="reviews__date">{ humanDate( review.date_created_gmt + 'Z' ) }</div>
 					</div>
-					<AutoDirection>
-						<div className="reviews__content">
-							{ decodeEntities( stripHTML( review.review ) ) }
+
+					{ review.verified && (
+						<div className="reviews__verified-label">
+							<Gridicon icon="checkmark-circle" size={ 18 } />
+							<span>{ translate( 'Verified product owner' ) }</span>
```

Comment:
Nit.  Could be shorter - i.e. "Product owner" or "Verified buyer" or "Bought product"

Reply:
Agreed it's a bit long. I'll go with `Verified buyer` for now, unless @kellychoffman or @jameskoster have any opinions on it.